### DISCUSSION TITLE
execution d'un programme

### DISCUSSION THREAD

**Discussion by Loubat39** [community/community#135504] [New to GitHub]
Bonjour, 
Etant très novice dans le domaine et autodidacte, je galère un peu sur l'exécution d'un programme.
cela fait plusieurs heur que j'essai de trouver comment démarrer un programme pris sur GitHub mais je comprend pas tout ..
si quelqu'un peut m'aider ...
merci

**Comment by ghostinhershell** [comment]
Hi @Loubat39, thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
